io_scene_godot/export_godot.py

```diff
@@ -121,9 +119,100 @@ def export_object(self, obj, parent_gd_node):
         # blender node exported as two parented node
         exported_node = exporter(self.escn_file, self.config, obj,
                                  parent_gd_node)
-
         self.bl_object_gd_node_map[obj] = exported_node
 
+        gds = None
```

Comment:
Move gdscript related change to another pull request, please.

Also I think it would be better to make another python file dedicated to gdscript related implementation. The main loop should be kept as small as possible